daemon/create.go

```diff
@@ -150,6 +150,14 @@ func (daemon *Daemon) generateSecurityOpt(ipcMode containertypes.IpcMode, pidMod
 
 		return label.DupSecOpt(c.ProcessLabel), nil
 	}
+	if pidContainer := pidMode.Container(); pidContainer != "" {
+		c, err := daemon.GetContainer(pidContainer)
+		if err != nil {
+			return nil, err
+		}
+
+		return label.DupSecOpt(c.ProcessLabel), nil
```

Comment:
If we have both `ipcMode.IsContainer()` and `pidMode.IsContainer()` are true and they both point to different containers with different labels we will never apply the second container labels.

I'm no selinux expert, but shouldn't those be merged somehow?